### ISSUE TITLE
Incorrect Perceptron default values for sklearn models

### ISSUE THREAD

**Issue by srcansiz** [fedbiomed/fedbiomed#498]
In GitLab by @ybouilla on Mar 20, 2023, 13:48

In Fed-BioMed, `Perceptron` model is currently created out of the `SGDClassifier` model by forcing the nature of the loss function (`loss=perceptron`). While mathematically correct, this way of doing things leads to some incorrect default values in `Perceptron` model, that are different to the default values of `SGDClassifier` (see [Perceptron](https://scikit-learn.org/stable/modules/generated/sklearn.linear_model.Perceptron.html) and [SGDClassifier](https://scikit-learn.org/stable/modules/generated/sklearn.linear_model.SGDClassifier.html) API documentation).

This issue has been open to address this issue: making sure initial default values are the same as the `Perceptron` model
